src/java.xml/share/classes/com/sun/org/apache/xerces/internal/dom/PSVIAttrNSImpl.java

```diff
@@ -60,16 +60,19 @@ public PSVIAttrNSImpl(CoreDocumentImpl ownerDocument, String namespaceURI,
     }
 
     /** attribute declaration */
+    @SuppressWarnings("serial") // Type of field is not Serializable
```

Comment:
For the classes that contain a number of these fields, do we want to put the annotation at the class level?

Reply:
It is more precise to just annotate the fields in question. Putting the annotation at the class level would disable all serial checking for the class so I think it is preferable to label the fields.